src/core/thread/meshcop_dataset_manager.cpp

```diff
@@ -155,7 +155,7 @@ void DatasetManager::HandleTimer(void)
     VerifyOrExit(mLocal.Compare(mNetwork) < 0,);
 
     Register();
-    mTimer.Start(1000);
+    // mTimer.Start(1000);
```

Comment:
The main reason that comments this line is that if we put active dataset to pending dataset, the device active dataset would not update immediately. Then the node would not stop c/as until the delay timer expires.

Not sure are there any other negative impacts here, if comment here.
But till my understanding, we do not need to depend this timer for the active dataset dissemination?